packages/heptagon/heptagon.1.03.01/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "gwenael.delaval@inria.fr"
+authors: "Gwenaël Delaval"
+homepage: "http://bzr.inria.fr"
+dev-repo: "git+ssh://scm.gforge.inria.fr//gitroot/heptagon/heptagon.git"
```

Comment:
The repository is indeed not yet intended to be publicly accessible. I remove this line.

Thanks.